### PR TITLE
Feat/login page

### PR DESCRIPTION
**Pull request by magsteen**
Created a basic login page with API calls and Vuex state for session/token.

### CODE REVIEW

File: frontend/src/components/LoginForm.vue [hunk 1]
```diff
@@ -0,0 +1,95 @@
+<template>
+  <form class="box" action="" v-on="submit">
+    <BaseInput
+      id="email"
+      label="Email"
+      v-model="email"
+      type="email"
+      class="login-form-item base-input-field"
+      placeholder="ola@nordman.no"
```
Comment (tomrsae) [inline frontend/src/components/LoginForm.vue]: This is never rendered, as [label is set as placeholder in BaseInput](https://github.com/StackScrubs/stuq/blob/dev/frontend/src/components/BaseInput.vue#L7) - might as well remove it.

File: frontend/src/service/SessionService.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { UserCredentials, InvalidCredentialsError } from "@/types/UserCredentials";
+import { Session } from "@/types/Session";
+import axios, { AxiosResponse } from "axios";
+
+const CONFIG = {
+    baseURL: "API AUTH SERVER URL" + "/login", //Set from .env
```
Comment (tomrsae) [inline frontend/src/service/SessionService.ts]: Think this should be resolved in this PR, so it's not forgotten about.
Unless of course this is blocked by something else, in which case resolve this comment yourself.

Comment (tomrsae) [inline frontend/src/service/SessionService.ts]: Endpoint will be called `session` from my understanding of [this change](https://github.com/StackScrubs/stuq/blob/5d414bfc875476fefea5c5f74744f3f091b02688/backend/src/main/java/com/github/stackscrubs/stuq/backend/controller/SessionController.java).

**Review by tomrsae — CHANGES_REQUESTED** [review]
*Great* job, left a few nitpicky comments :d

File: frontend/src/service/SessionService.ts (hunk 2 above)
Comment (tomrsae) [reply frontend/src/service/SessionService.ts]: Referring to the `//Set from .env` comment

File: frontend/src/components/LoginForm.vue (hunk 1 above)
Comment (magsteen) [reply frontend/src/components/LoginForm.vue]: :+1:

**Review by tomrsae — APPROVED** [review]
Great job :D

**Pull request merged**